2024.py

```diff
@@ -45,7 +45,11 @@
 
 # Identify the directory with text files to explore
 workingDir = os.getcwd()
-CollPath = os.path.join(workingDir, 'textCollection')
+# CollPath = os.path.join(workingDir, 'textCollection')
+# 2024-02-07 ebb: You need to have a folder with input text files.
+# I just created one in your repo, and copied the two text files you had into it.
+# I named it "input-files" so now I point the CollPath to that folder:
+CollPath = os.path.join(workingDir, 'input-files')
```

Comment:
@TylerDollard Here's where we  need to read in your files. Point it to a folder inside your working directory!  (I created an input-files folder and just tossed a couple of your text files here to read.)